### ISSUE TITLE
vcf-split breaks sorting

### ISSUE THREAD

**Issue by fgvieira** [rust-bio/rust-bio-tools#229]
`vcf-split` keeps breakends together, but I thought it would just keep them in the same file. In practice, it moves the records around and breaks the file sorting.

So, for example, records:
```
III     3226364 MantaDEL:78744:0:0:0:0:0
III     3228674 MantaBND:78991:0:1:1:0:0:1
III     3228676 MantaBND:78991:0:1:0:1:0:1
III     3228678 MantaINS:78991:0:0:0:0:0
III     3228681 MantaDEL:78991:0:0:0:0:1
III     3228782 MantaBND:78991:0:1:0:1:0:0
III     3228783 MantaBND:78991:0:1:1:0:0:0
```

become:

```
III     3226364 MantaDEL:78744:0:0:0:0:0
III     3228678 MantaINS:78991:0:0:0:0:0
III     3228681 MantaDEL:78991:0:0:0:0:1
III     3228674 MantaBND:78991:0:1:1:0:0:1
III     3228676 MantaBND:78991:0:1:0:1:0:1
III     3228782 MantaBND:78991:0:1:0:1:0:0
III     3228783 MantaBND:78991:0:1:1:0:0:0
```

Is there any reason why this behavior?
Would it make sense to add an option to keep the records sorted?

thanks,

**Comment by johanneskoester** [comment]
Good catch. This is of course not intended.

**Comment by tedil** [comment]
Closing because this has been fixed with #244
